tests/e2e/travel.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '@playwright/test';
+import { TravelPage } from 'e2e-tests/pom/travel.page';
+
+let travelPage : TravelPage;
+
+test.beforeEach('setup', async ({page}) => {
+    travelPage = new TravelPage(page);
+
+  await travelPage.goTo();
+});
+
+test('Validate card number (Visa/Master)', async ({ page }) => {
```

Comment:
From test name i understand you will be checking if card has correct number on it, but checking deeper - methods are checking that flight can be booked using visa/mastercard (booking confirmation received) - consider changing test name to be more descriptive